audbcards/core/datacard.py

```diff
@@ -222,6 +350,7 @@ def _expand_dataset(
                 player = self.player(example)
                 dataset['player'] = player
                 dataset['example'] = example
+        dataset['file_duration_distribution'] = self.file_duration_distribution
         return dataset
 
     def _render_template(self):
```

Comment:
```suggestion
    def _render_template(self) -> str:
```
All other type hints seem complete, except this one.